app/views/admin/products_v3/_filters.html.haml

```diff
@@ -4,10 +4,10 @@
       = text_field_tag :search_term, search_term, placeholder: t('.search_products')
   .producers
     .label= t('.producers.label')
-    = select_tag :producer_id, options_for_select(producer_options, producer_id), include_blank: t('.all_producers')
+    = select_tag :producer_id, options_for_select(producer_options, producer_id), include_blank: t('.all_producers'), "data-controller": "tom-select", "data-tom-select-options-value": '{ "plugins": [] }', class: "fullwidth"
```

Comment:
Perhaps we want to make a `tom_select_tag` helper, with predefined options for "single", etc. We can style it with a class.